.github/workflows/CI.yaml

```diff
@@ -295,3 +295,41 @@ jobs:
           version: "dev"
           klt-repo: ${{ github.workspace }}
           token: ${{ secrets.KEPTN_SANDBOX_BOT_TOKEN }}
+
+  helm_charts_build:
+    name: Publish helm chart changes to charts repo
+    if: github.event_name == 'push' && needs.prepare_ci_run.outputs.NON_FORKED_AND_NON_ROBOT_RUN == 'true'
```

Comment:
this would create a PR for each commit, or do I miss something?

Reply:
yes, the pipeline tries to create a PR for each commit to main, but actually create it only if there were any changes in helm/chart/ directory, so only when something in the helm-charts has changed